### DISCUSSION TITLE
SKr V1.4, TFT and SD card conflict

### DISCUSSION THREAD

**Discussion by 66Drew66** [community/community#76957] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, it's my first time on here so if I'm on the wrong page then I do apologise.
I've installed a Bigtreetech SKR V1.4 Turbo main board in my creality cr10s 3d printer. 
I'm using Marlin 2.1.2.1 and the related bugfix 2.1.X

I have two problems,
1. TFT screen is showing the error message "No Printer"
2.  The sd card reader isn't recognising any media. 

From a post on another site I understand that the problem is down to both the SD card reader and the TFT screen using the same port.
Unfortunately the solution to the issue posted there doesn't seam to be available in my config adv.h file.
The solution was as follows.

_**Hi, With the TFT screen you have effectively running 2 SD card readers. Have you tried setting the following line in Configuration_adv.h to ONBOARD rather than LCD, this should direct to the onboard card reader instead of the Display based reader. Worked for me. Apologies if this is already commented and missed it. Best Boothy

Configuration_adv.h Line 1293 approx

#define SDCARD_CONNECTION ONBOARD

Edit: You should still be able to run the board firmware via the display SD card reader also (I think) you will just have to have the display set to Simulated mode to see the reboot rather than touch display. The firmware will probably still upgrade from the onboard reader but may not just read files if that makes sense. I had the same scenario and it worked OK for me and fixed the issue._**

I've attached the two Marlin firmware files the SD card reader setting start somewhere around the 1500's in bugfix 2.1.x

Thanks.

**Comment by Akash1134** [comment]
Hi @66Drew66 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
